src/tab/Tab.tsx

```diff
@@ -6,15 +6,15 @@ import classNames from "classnames";
 import TabHeaderItem from "./header/item/TabHeaderItem";
 import List from "../list/List";
 
-export type TabItem = {
-  id: string;
+export type TabItem<ID extends string | number = string | number> = {
```

Comment:
Just wonder, if the ID can be a string or number, should we extend it with these types? Wouldn't be okay like this? 🤔 
```suggestion
export type TabItem<ID = string | number> = {
```

Reply:
With the current version the type of ID can only string or number (same as your suggestion), but if you don't specify a type argument, the default type will be one of those two.